src/llmcompressor/pipelines/sequential/pipeline.py

```diff
@@ -14,6 +14,7 @@
     get_sequential_targets,
     trace_subgraphs,
 )
+from llmcompressor.utils.dev import infer_model_device
```

Comment:
same ask here, do we expect users to be using this anywhere? if so importing from a `.dev` might be confusing

Reply:
In the future it's most robust to have this be imported in the example scripts, so I can raise this import path